packages/create-react-app/index.js

```diff
@@ -65,44 +67,78 @@ var program = commander
   .version(require('./package.json').version)
   .arguments('<project-directory>')
   .usage(chalk.green('<project-directory>') + ' [options]')
-  .action(function (name) {
+  .action(function(name) {
     projectName = name;
   })
   .option('--verbose', 'print additional logs')
-  .option('--scripts-version <alternative-package>', 'use a non-standard version of react-scripts')
+  .option(
+    '--scripts-version <alternative-package>',
+    'use a non-standard version of react-scripts'
+  )
   .allowUnknownOption()
-  .on('--help', function () {
-    console.log('    Only ' + chalk.green('<project-directory>') + ' is required.');
+  .on('--help', function() {
+    console.log(
+      '    Only ' + chalk.green('<project-directory>') + ' is required.'
+    );
     console.log();
-    console.log('    A custom ' + chalk.cyan('--scripts-version') + ' can be one of:');
+    console.log(
+      '    A custom ' + chalk.cyan('--scripts-version') + ' can be one of:'
+    );
     console.log('      - a specific npm version: ' + chalk.green('0.8.2'));
-    console.log('      - a custom fork published on npm: ' + chalk.green('my-react-scripts'));
-    console.log('      - a .tgz archive: ' + chalk.green('https://mysite.com/my-react-scripts-0.8.2.tgz'));
-    console.log('    It is not needed unless you specifically want to use a fork.');
+    console.log(
+      '      - a custom fork published on npm: ' +
+        chalk.green('my-react-scripts')
+    );
+    console.log(
+      '      - a .tgz archive: ' +
+        chalk.green('https://mysite.com/my-react-scripts-0.8.2.tgz')
+    );
+    console.log(
+      '    It is not needed unless you specifically want to use a fork.'
+    );
     console.log();
-    console.log('    If you have any problems, do not hesitate to file an issue:');
-    console.log('      ' + chalk.cyan('https://github.com/facebookincubator/create-react-app/issues/new'));
+    console.log(
+      '    If you have any problems, do not hesitate to file an issue:'
+    );
+    console.log(
+      '      ' +
+        chalk.cyan(
+          'https://github.com/facebookincubator/create-react-app/issues/new'
+        )
+    );
```

Comment:
If you want all this code to stay on the same line I'd recommend either string templates or increasing the `printWidth` option to something larger, say 100. Honestly it doesn't look ugly to me, it's exactly what I'd expect after using prettier for a while. I'd be surprised if it didn't format this! But if you care about how it looks, string templates are the way to go.

One small thing here is that we could fix the indentation, it doesn't need to indent the sub-expressions, which is the only ugly thing I see.